src/styles/style.css

```diff
@@ -24,12 +24,87 @@ body {
   margin: 0 auto;
 }
 
+.no-aside-container {
+  justify-content: space-between;
+  width: 90%;
+  max-width: 1100px;
+  margin: 0 auto;
+}
+
+.no-aside {
+  max-width: 80ch;
+}
+
+ul {
+  list-style-position: inside;
+  list-style: none;
+  margin-top: 0.65rem;
+}
+
+ul li {
+  margin-top: 0.433rem;
+}
+
+/*
+ * TYPOGRAPHY
+ */
+
+h1 {
+  font-size: 2.534rem;
+  line-height: 128.25%;
+}
+
+h2 {
+  font-size: 2.253rem;
+  line-height: 144.282%;
+}
+
+h3 {
+  font-size: 2.002rem;
+  line-height: 162.317%;
+}
+
+h4 {
+  font-size: 1.78rem;
+  line-height: 182.606%;
+}
+
+h5 {
+  font-size: 1.582rem;
+  line-height: 102.716%;
+}
+
+h6 {
+  font-size: 1.406rem;
+  line-height: 115.555%;
+}
+
+p {
+  font-size: 1.25rem;
+  line-height: 130%;
+  margin-top: 0.26rem;
+}
+
+p + p {
+  font-size: 1.25rem;
+  line-height: 130%;
+  margin-top: 1rem;
+}
+
+#content h2,
```

Comment:
Just a heads up, I j=changed the id on the `main` tag to be mc in my `publish-content`. Wanted to avoid an english only id.

Reply:
so:
 id="content" => id="mc" ?

Reply:
Correct